root/static/scripts/edit/URLCleanup.js

```diff
@@ -4544,6 +4543,38 @@ const CLEANUPS: CleanupEntries = {
     ],
     restrict: [LINK_TYPES.otherdatabases],
   },
+  'ototoy': {
+    match: new RegExp('^(https?://)?([^/]+\\.)?ototoy\\.jp', 'i'),
+    restrict: [LINK_TYPES.downloadpurchase],
+    clean: function (url) {
+      return url.replace(/^(?:https:\/\/)?(?:www\.)?ototoy\.jp\/(labels|_\/default\/[ap])\/(\d+).*$/, 'https://ototoy.jp/$1/$2');
```

Comment:
You allow for `http` in `match` but do not actually clean up non-https links here, it seems.

Reply:
If I understand correctly, this is just about the missing `?` after `https`:

```suggestion
      return url.replace(/^(?:https?:\/\/)?(?:www\.)?ototoy\.jp\/(labels|_\/default\/[ap])\/(\d+).*$/, 'https://ototoy.jp/$1/$2');
```

(Compare to the regular expression in `match`.)